modules/system/classes/SystemController.php

```diff
@@ -27,7 +27,7 @@ public function combine($name)
     {
         try {
             if (!strpos($name, '-')) {
-                throw new ApplicationException(Lang::get('system::lang.combiner.not_found', ['name' => $name]));
+                return Response::make('/* '.e(Lang::get('system::lang.combiner.not_found', ['name' => $name])).' */', 404);
```

Comment:
Can you update the docblock `@return string` to say `@return Response` for both changed methods please?